explorer/src/tests/webdriver/wdio.conf.js

```diff
@@ -0,0 +1,190 @@
+const url = require('./helpers/urls')
+const ENV = process.env.ENV
+
+if(!ENV ||  !['local', 'devfeature', 'prod'].includes(ENV))
+{
+    console.log("please input the correct env, check the urls module for env urls")
+    console.log("exiting")
+    process.exit()
+}
+
+exports.config = {
+    specs: [
+        './tests/specs/**/*.js'
+    ],
+    // Patterns to exclude.
+    exclude: [
+        // 'path/to/excluded/files'
+    ],
+    maxInstances: 10,
+    capabilities: [{
+        maxInstances: 5,
+        //
+        browserName: 'chrome',
```

Comment:
Might be overkill given so much is MUI & heavily tested already, but is it possible to add other browsers to this? Firefox? Edge?

Reply:
Sure, it's all configurable. There's another config file that will be run specifically for Github actions. This was just the boilerplate out the box on set up.